docs/sources/structure/topic-types/task/index.md

```diff
@@ -25,7 +25,8 @@ A _task_ topic includes the following elements:
 
 - **Introduction:** Provide an introduction that explains why the end user should care about the task.
 
-  There might be conceptual material in this section of a task topic. Limit conceptual information to only what's relevant to the task at hand.
+  There might be conceptual material in this section of a task topic.
+  Limit conceptual information to only what's relevant to the task at hand.
```

Comment:
This is fine, but I would have written it differently. I think the second sentence is subsidiary, but it should start with "If the topic needs concepts explained, then limit..."